advanced/Scripts/piholeDebug.sh

```diff
@@ -1348,30 +1348,33 @@ check_database_integrity() {
 # Show a text spinner during a long process run
 #
 spinner(){
-    local PID=$!  # PID of the most recent background process
-    local spin="/-\|"
-    local start=0
-    local elapsed=0
-    local i=1
-
-    start=$(date +%s) # Start the counter
-
-    tput civis > /dev/tty # Hide the cursor
-    trap 'tput cnorm > /dev/tty' EXIT # ensures cursor is visible again, in case of premature exit
-
-    while [ -d /proc/$PID ]; do
-        elapsed=$(( $(date +%s) - start ))
-        # print the spinner only on screen (tty) - use hours only if needed
-        if [ "$elapsed" -lt 3600 ]; then
-            printf "\r${spin:i++%${#spin}:1} %02d:%02d" $((elapsed/60)) $((elapsed%60)) >"$(tty)"
-        else
-            printf "\r${spin:i++%${#spin}:1} %02d:%02d:%02d" $((elapsed/3600)) $(((elapsed/60)%60)) $((elapsed%60)) >"$(tty)"
-        fi
-        sleep 0.25
-    done
+    # Show the spinner only if there is a tty
+    if tty -s; then
+        local PID=$!  # PID of the most recent background process
+        local spin="/-\|"
+        local start=0
+        local elapsed=0
+        local i=1
+
+        start=$(date +%s) # Start the counter
```

Comment:
Just a style thing but can you move the inline comments to their own line?

Reply:
Done